src/Quidjibo.DataProtection/Quidjibo.DataProtection.csproj

```diff
@@ -13,10 +13,14 @@
     <Authors>Smiggleworth</Authors>
     <Company>Smiggleworth</Company>
     <Version>1.0.0</Version>
+    <LangVersion>7.3</LangVersion>
```

Comment:
We need to bump the language version requirement for some of the ref-struct functionality for `Span<T>`.

Reply:
Deferring to @smiggleworth

Reply:
looks good to me.